homeassistant/components/tessie/strings.json

```
@@ -529,5 +529,35 @@
     "command_failed": {
       "message": "Command failed, {message}"
     }
+  },
+  "services": {
+    "navigation_request": {
+      "name": "Set navigation destination",
+      "description": "Set navigation to a destination.",
```

Action descriptions should do more than more or less repeat the name of the action. Also they should use third person singular to sound descriptive. Suggestion (feel free to change in some other way):

```suggestion
      "description": "Sends a destination address to the onboard navigation system.",
```